toast-tk-agent-api/src/main/java/io/toast/tk/agent/config/WebConfigProvider.java

```diff
@@ -0,0 +1,58 @@
+package io.toast.tk.agent.config;
+
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.Properties;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import com.google.inject.Provider;
+
+
+
+public class WebConfigProvider implements Provider<WebConfig> {
+	
+	private static final Logger LOG = LogManager.getLogger(WebConfigProvider.class);
+
+	private WebConfig webConfig;
+
+	
+	public static final String TOAST_TEST_WEB_INIT_RECORDING_URL = "toast.web.recording.url";
+
+	public static final String TOAST_CHROMEDRIVER_PATH = "toast.chromedriver.path";
+	
+	public static final String TOAST_TEST_WEB_APP_URL = "toast.webapp.url";
+
+	public static final String TOAST_API_KEY = "toast.api.key";
+
+	public static final String TOAST_PLUGIN_DIR = "toast.api.key";
+
+	public WebConfigProvider() {
+		super();
+	}
+
+	private void initConfig() throws NullPointerException {
+		String userHomepath = WebConfig.getToastHome() + "/";
+		Properties p = null;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Remove this useless assignment to local variable "p". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1854)